osmcb-catalog-Test001.xml

```diff
@@ -0,0 +1,104 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<bundle version="1" name="Netherlands" outputFormat="BCOpenCPN">
+    <Layer name="L08">
+        <Map mapSource="AH OpenSeaMap - Mapnik" zoom="8" lrc="N55°46'35,7&quot; E15°28'27,3&quot;" maxTileCoordinate="35585/23297" minTileCoordinate="32768/20480" name="L08 000-000" ulc="N46°04'09,9&quot; E0°00'00,0&quot;"/>
+        <Map mapSource="AH OpenSeaMap - Mapnik" zoom="8" lrc="N55°46'35,7&quot; E15°28'27,3&quot;" maxTileCoordinate="35585/23297" minTileCoordinate="32768/20480" name="L08 001-000" ulc="N46°04'09,9&quot; E0°00'00,0&quot;"/>
```

Comment:
Since it is 'originally' from the chart-designer, which is written in Java, it has been easy to implement there.
I can change the lat/lon format with no problem at all to dd.dddddd. It will be done in the next commit to the chart designer.

Alexej